jmeter-java-dsl-dashboard/src/test/java/us/abstracta/jmeter/javadsl/dashboard/DashboardVisualizerTest.java

```diff
@@ -34,5 +45,21 @@ public void shouldDisplayGraphsAndSummaryWhenRunTestPlanWithDashboard(TestInfo t
         testInfo
     );
   }
+    @Test
+    public void testShowTestElementGui() {
+        DashboardVisualizer dashboardVisualizer = new DashboardVisualizer();
+        JPanel panel = new JPanel();
+        Runnable closeListener = () -> System.out.println("Closing");
+        dashboardVisualizer.showTestElementGui(panel, closeListener);
+    }
+    @Test
+    public void testShowInGui() {
```

Comment:
I think this in general this is a good test to add, that allows users to learn about one particular aspect of the contract of DashboardVisualizer. But, I do think it requires some changes:
* The name of the test should follow the naming convention we have followed in other tests and should describe the purpose of the test. For example: `shouldThrowUnsupportedOperationWhenShowInGuiOnDasbhardVisualizer`
* In general avoid using System.out, and prefer using logs, but in this case is more important to fail the test if the expected condition is not met (which doesn't happen in current implementation of the test). I good practice to test for exceptions in JUnit 5 is using assertThrows: https://howtodoinjava.com/junit5/expected-exception-example/